Gemfile

```diff
@@ -46,28 +46,28 @@ gem 'select2-rails'
 gem 'nokogiri', '1.6.7.2'
 
 group :development do
-  gem "binding_of_caller"
-  gem "better_errors"
-  gem "rubocop"
+  gem 'binding_of_caller'
+  gem 'better_errors'
   gem 'web-console', '2.1.3'
   gem 'capistrano', '~> 3.1'
   gem 'capistrano-rails', '~> 1.1'
   # gem 'capistrano3-delayed-job', '~> 1.0'
   gem 'airbrussh', require: false
-  gem "quiet_assets"
+  gem 'quiet_assets'
 end
 
 group :development, :test do
-  gem "rspec-rails", '~> 3.4.0'
-  gem "capybara", '~> 2.4.4'
-  gem "byebug"
+  gem 'rspec-rails', '~> 3.4.0'
+  gem 'capybara', '~> 2.4.4'
+  gem 'byebug'
+  gem 'rubocop', '~> 0.40'
```

Comment:
Aham, so I guess that adding the gem in testing environment.. HoundCI will use that exact version :+1:
Didn't know about that..